src/main/java/utilities/UtilitiesFunctions.java

```diff
@@ -0,0 +1,42 @@
+package utilities;
+
+import managers.Context;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Set;
+
+public class UtilitiesFunctions {
+
+    private static final Logger log = LogManager.getLogger(Logger.class.getName());
+
+    public static WebElement getWebElement(String xpath) {
+        return Context.driverManager.getDriver().findElement(By.xpath(xpath));
+    }
+
+    public static void switchToOtherTab() {
+        WebDriver driver = Context.driverManager.getDriver();
+
+        try {
+            Thread.sleep(2000);
+        } catch (java.lang.InterruptedException e) {
+            log.error(e);
+        }
+
+        String currentTab = driver.getWindowHandle();
+        Set<String> handles = driver.getWindowHandles();
+        if (handles.size() < 2) {
+            log.error("There is only one tab opened");
+        }
+
+        for (String tab : handles) {
+            if (!tab.equals(currentTab)) {
+                driver.switchTo().window(tab);
```

Comment:
should we also have a method that closes the tab that you left?

Reply:
Also, what about the case when there is more than 2 tabs open? What is the behaviour you're expecting?

Reply:
We could add methods for closing/opening specific tabs to to your right/left or with a specific number/phrase - I just don't use that in my projects and I wasn't creating anything new for this PR

If there is more than 2 tabs, then this method will just switch between the 2 first ones. It's pretty much the idea behind it, it's simple, doesn't take any arguments etc. - just switches to another tab than you're already on, which is useful for many simple test cases

Reply:
I think it will switch to second, than third, until the driver reaches the last tab (because of "for tabs" loop)

Reply:
Driver will never reach 3rd tab because 2 tabs can't be active at the same time. So it doesn't matter if there are 2, or more tabs opened